### PR TITLE
Migrate all master-gce jobs from jenkins to prow

### PR DESCRIPTION
**Pull request by krzyzacy**
tot is verified working, so let's take stuff off jenkins :-)

left gce-enormous, and 1.6/1.7 scalability jobs, renamed them to match rest of the jobs.

sending another PR for gke ones.

### CODE REVIEW

**Comment by rmmh** [comment]
@spxtr do we need to expand the prow builder pool to support this?

**Comment by rmmh** [comment]
I'm slightly worried about our lack of prioritization and running out of resources if CI and PR jobs start fighting.

**Comment by krzyzacy** [comment]
@rmmh current CPU usage is ~40%, should be fine.

**Comment by krzyzacy** [comment]
odd that I only can see 40 nodes from k8s dashboard, should have 50 though

**Comment by krzyzacy** [comment]
deployed, watching...

File: jobs/config.json [hunk 1]
```diff
@@ -2767,33 +2806,36 @@
...
       "--env-file=jobs/platform/gce.env",
-      "--env-file=jobs/ci-kubernetes-e2e-gci-gce-scalability-release-1-7.env",
-      "--extract=ci/latest-1.7",
-      "--cluster=e2e-scalability-1-7",
-      "--timeout=120m"
+      "--env-file=jobs/ci-kubernetes-e2e-gci-gce-scalability-release-1-6.env",
+      "--cluster=e2e-scalability-1-6",
+      "--timeout=120m",
+      "--extract=ci/latest-1.6",
+      "--mode=local"
```
Comment (ixdy) [inline jobs/config.json]: this is still running on Jenkins, so should not be `--mode=local`.

File: jobs/config.json [hunk 2]
```diff
@@ -2767,33 +2806,36 @@
...
     "args": [
       "--env-file=jobs/platform/gce.env",
-      "--env-file=jobs/ci-kubernetes-e2e-gci-gce-scalability-release-1.6.env",
-      "--cluster=e2e-scalability-1-6",
+      "--env-file=jobs/ci-kubernetes-e2e-gci-gce-scalability-release-1-7.env",
+      "--extract=ci/latest-1.7",
+      "--cluster=e2e-scalability-1-7",
       "--timeout=120m",
-      "--extract=ci/latest-1.6"
+      "--mode=local"
```
Comment (ixdy) [inline jobs/config.json]: same. https://github.com/kubernetes/kubernetes/issues/48523

**Pull request merged**